### ISSUE TITLE
opacity on emoji

### ISSUE THREAD

**Issue by gosoccerboy5** [jeffalo/ocular#71]
**Describe the bug**
I cant see the emoji

**To Reproduce**
1. Go to https://ocular.jeffalo.net/post/5293197/
2. Click on the emoji react thingy
3. Hover on an emoji reaction
4. See error - the emoji disappears

**Expected behavior**
I should see it, but there should still be distinct behavior on hover

**Screenshots**
<img width="257" alt="Screen Shot 2021-05-30 at 1 28 51 PM" src="https://user-images.githubusercontent.com/82768218/120115815-22791280-c14b-11eb-81a4-fb03f93cbbbc.png">

**Desktop (please complete the following information):**
 - OS: mac
 - Browser firefox
 - Version 11.3

To fix:
change
```css
.reaction-button-popover[data-v-3692234e]:hover {
 opacity:1%
}
```
to something like
```css
.reaction-button-popover[data-v-3692234e]:hover {
 opacity:50%
}
```

**Comment by jeffalo** [comment]
alright and now here is the mind bending part.. https://github.com/jeffalo/ocular/blob/b95a96715137c267c3133ee505aefdb495a1d371/components/ReactionButtons.vue#L140-L142

it says 75%.. how did it become 1% in production!!???

**Comment by gosoccerboy5** [comment]
Yay, have fun debugging..

**Comment by jeffalo** [comment]
fixed somehow d95d608a7a415b6e28bc28cb7b4d37ca3beec1fe

**Comment by gosoccerboy5** [comment]
ayy it works
